### PR TITLE
Success Rate Logging to Tensorboard

### PR DESCRIPTION
**Pull request by peasant98**
As per the title - log the success rate to tensorboard instead of just a `logging.info`.

### CODE REVIEW

File: pfrl/experiments/evaluator.py [hunk 1]
```diff
@@ -363,14 +366,28 @@ def eval_performance(
...
+
+    else:
+        stats = dict(
+            episodes=len(scores),
+            mean=statistics.mean(scores),
+            median=statistics.median(scores),
+            stdev=statistics.stdev(scores) if len(scores) >= 2 else 0.0,
+            max=np.max(scores),
+            min=np.min(scores),
+        )
```
Comment (watakandai) [inline pfrl/experiments/evaluator.py:390]: Another option is to add switch to `batch_run_evaluation_episodes` and `run_evaluation_episodes` to return scores AND success_rate. That way the code might be a bit cleaner.

**Comment by watakandai** [comment]
Does this PR have a feature to record a video at every evaluation step?

**Comment by watakandai** [comment]
> Does this PR have a feature to record a video at every evaluation step?

I found it in the entropy-fix branch.

**Comment by peasant98** [comment]
cool

**Pull request merged**